specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Storage/Storage.tsp

```diff
@@ -168,6 +168,9 @@ namespace Microsoft.EventGrid.SystemEvents {
 
     /** Execution statistics of a specific policy action in a Blob Management cycle. */
     tierToArchiveSummary: StorageLifecyclePolicyActionSummaryDetail;
+
+    /** Execution statistics of a specific policy action in a Blob Management cycle. */
```

Comment:
Should the descriptions include any information that is specific to this property? It looks like the descriptions are identical for each of the properties having a type of StorageLifecyclePolicyActionSummaryDetail.

Reply:
Thank you for the thought, this would help for clarity. But I think for now, the action name itself emphasizes its description.